INSTALL.md

```diff
@@ -130,11 +132,72 @@ After confirming that it matches, type `y` (for yes) and the installation should
 
 ### Qubes OS
 
-> :warning: This section is for the experimental native Qubes support. If you want
-> to try out the stable Dangerzone version (which uses containers instead of virtual machines for isolation),
-> please follow the Fedora or Debian instructions and adapt them as needed.
+> [!WARNING]
+> This section is for the beta version of native Qubes support. If you
+> want to try out the stable Dangerzone version (which uses containers instead
+> of virtual machines for isolation), please follow the Fedora or Debian
+> instructions and adapt them as needed.
+
+> [!IMPORTANT]
+> This section will install Dangerzone in your **default template**
+> (`fedora-38` as of writing this). If you want to install it in a different
+> one, make sure to replace `fedora-38` with the template of your choice.
+
+The following steps must be completed once. Make sure you run them in the
+specified qubes.
+
+Overview of the qubes you'll create:
+
+| qube         |   type   | purpose |
+|--------------|----------|---------|
+| dz-dvm       | app qube | offline diposable template for performing conversions |
+
+#### In `dom0`:
+
+Create a **disposable**, offline app qube (`dz-dvm`), based on your default
+template. This will be the qube where the documents will be sanitized:
+
+```
+qvm-create --class AppVM --label red --template fedora-38 \
+    --prop netvm="" --prop template_for_dispvms=True \
+    dz-dvm
+```
+
+Add an RPC policy (`/etc/qubes/policy.d/50-dangerzone.policy`) that will
+allow launching a disposable qube (`dz-dvm`) when Dangerzone converts a
+document, with the following contents:
+
+```
+dz.Convert         *       @anyvm       @dispvm:dz-dvm  allow
+```
+
+#### In the `fedora-38` template
+
+Install Dangerzone:
+
+```
+sudo dnf config-manager --add-repo=https://packages.freedom.press/yum-tools-prod/dangerzone/dangerzone.repo
+sudo dnf install dangerzone-qubes
+```
+
+> [!NOTE]
+> The `dangerzone-qubes` package by default includes OCR support for the 10 most
+> common languages: English, Chinese (Mandarin), Hindi, Spanish, French, Arabic
+> (Standard), Bengali, Portuguese, Russian, and Urdu. If you want to add OCR
+> support for another language, simply install the corresponding
+> `tesseract-langpack-<lang>` package in your template.
```

Comment:
Now that all the available languages were added, this needs some updating.

Reply:
Fixed in f7b46f18a576fb2fa4cff63390d96349bb592d42.